tower-balance/src/p2c/make.rs

```diff
@@ -1,25 +1,32 @@
 use super::Balance;
 use futures::{try_ready, Future, Poll};
 use rand::{rngs::SmallRng, FromEntropy};
+use std::marker::PhantomData;
 use tower_discover::Discover;
 use tower_service::Service;
 
 /// Makes `Balancer`s given an inner service that makes `Discover`s.
 #[derive(Clone, Debug)]
-pub struct BalanceMake<S> {
+pub struct BalanceMake<S, Req> {
     inner: S,
     rng: SmallRng,
+    _marker: PhantomData<fn(Req)>,
```

Comment:
Why `fn(Req)` over just `Req`?

Reply:
We do this here (and elsewhere) so that `Req` doesn't get the `Sync/Send/'static` requirements of the Service.  I.e. `Req` need not be `Send` just because `Service` is, etc.

I'm pretty sure I picked up this habit from @seanmonstar...